hudi-client/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -51,6 +52,11 @@
   // TODO: On by default. Once stable, we will remove the other mode.
   public static final String BLOOM_INDEX_BUCKETIZED_CHECKING_PROP = "hoodie.bloom.index.bucketized.checking";
   public static final String DEFAULT_BLOOM_INDEX_BUCKETIZED_CHECKING = "true";
+  public static final String BLOOM_INDEX_FILTER_TYPE_PROP = "hoodie.bloom.index.filter.type.prop";
```

Comment:
remove `.prop` from property name

Reply:
There are lot of properties with suffix "PROP" 
BLOOM_INDEX_TREE_BASED_FILTER_PROP
BLOOM_INDEX_BUCKETIZED_CHECKING_PROP
and hence I followed the same